Doc/whatsnew/3.9.rst

```diff
@@ -138,4 +138,10 @@ Porting to Python 3.9
 This section lists previously described changes and other bugfixes
 that may require changes to your code.
 
+Changes in Python behavior
+--------------------------
 
+* :func:`open`, :func:`io.open`, :func:`codecs.open` and
+  :class:`fileinput.FileInput` no longer accept ``U`` ("universal newline") in
+  the file mode. This flag was deprecated since Python 3.3.
```

Comment:
Would be nice to say what is the replacement (the `newline` argument in `io.open()`).

Reply:
done